core/pn_counter.go

```diff
@@ -54,65 +54,65 @@ type PNCounter interface {
 	IDistributedObject
 
 	// Get returns the current value of the counter.
-	// It returns HazelcastNoDataMemberInClusterError if the cluster does not contain any data members.
-	// It returns HazelcastUnsupportedOperationError if the cluster version is less  than 3.10.
-	// It returns HazelcastConsistencyLostError if the session guarantees have been lost.
+	// Get returns HazelcastNoDataMemberInClusterError if the cluster does not contain any data members,
+	// HazelcastUnsupportedOperationError if the cluster version is less  than 3.10,
+	// HazelcastConsistencyLostError if the session guarantees have been lost.
 	Get() (currentValue int64, err error)
 
 	// GetAndAdd adds the given value to the current value.
-	// It returns the previous value.
-	// It returns HazelcastNoDataMemberInClusterError if the cluster does not contain any data members.
-	// It returns HazelcastUnsupportedOperationError if the cluster version is less  than 3.10.
-	// It returns HazelcastConsistencyLostError if the session guarantees have been lost.
+	// GetAndAdd returns the previous value.
```

Comment:
These changes are unnecessary. `It` references `GetAndAdd `.
Look at this. This is a built-in method and uses `it` while referencing method name.

````
// grow grows the buffer to guarantee space for n more bytes.
// It returns the index where bytes should be written.
// If the buffer can't grow it will panic with ErrTooLarge.
func (b *Buffer) grow(n int) int {
````

There are many like this. This is not about `Go` convention.

Reply:
Agreed.